Graphs/LCABinaryLifting.js

```diff
@@ -6,47 +6,21 @@
  * Tutorial on Lowest Common Ancestor: https://www.geeksforgeeks.org/lca-in-a-tree-using-binary-lifting-technique
  */
 
+import { BinaryLifting } from './BinaryLifting'
+
 class LCABinaryLifting {
   constructor (root, tree) {
-    this.root = root
-    this.connections = new Map()
-    this.up = new Map() // up[node][i] stores the 2^i-th  parent of node
+    this.binaryLifting = new BinaryLifting(root, tree)
```

Comment:
This is rather dirty. Why not use inheritance here?

Reply:
Made use of inheritance in code